src/applications/disability-benefits/all-claims/reviewErrors.js

```diff
@@ -0,0 +1,37 @@
+import numberToWords from 'platform/forms-system/src/js/utilities/data/numberToWords';
+
+// Link text for review & submit page errors
+// key = "name" from `form.formErrors.errors`
+export default {
+  servicePeriods:
```

Comment:
This needs documentation for other folks to follow later. Could you write a few words in like `src/platform/forms-system/docs/` or something? (I know that doesn't exist right now, but we need _something_ and I'm not sure where is the right place for that right now.)